package.json

```diff
@@ -8,7 +8,7 @@
     "dev:server": "cross-env NODE_ENV=development npm run ts:exec server/index.ts",
     "dev": "nodemon",
     "docker:db": "docker-compose exec pg psql -U postgres -d dev-directory",
-    "docker:deps": "docker-compose up",
+    "docker:deps": "docker-compose up -d",
```

Comment:
@timmyichen @Caleb-Cohen Was this intentional and meant to be a part of this PR?

Reply:
yep - for most people, they won't have to dig through SQL logs, and so we want the default happy path to just be "get the stack running" without needing to open a separate terminal for it.  if logs are needed, they can either use `docker-compose up` or look at the docker container logs via the desktop app - and we should probably add docs for this, but the default path should imo still aim to get it up and running as simply as possible

Reply:
Fair, just wasn't sure it should be in this PR... either way, works for me. 👍🏼